microservices/document_manager/backend/.env-example

```diff
@@ -0,0 +1,21 @@
+DB_HOST=postgres
```

Comment:
Update .env-example with correct credentials so peeps dont have to hunt down docker-compose or not get confused that this is using the same schema as nris

Reply:
thanks.